pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -185,8 +185,17 @@ func (pl *VolumeBinding) PreFilter(ctx context.Context, state *framework.CycleSt
 		status.AppendReason("pod has unbound immediate PersistentVolumeClaims")
 		return nil, status
 	}
+	// Attempt to reduce down the number of nodes to consider in subsequent scheduling stages if pod has bound claims.
+	var result *framework.PreFilterResult
+	nodes, err := pl.Binder.GetEligibleNodes(boundClaims)
+	if err == nil && len(nodes) > 0 {
```

Comment:
Given you don't rely on the `err`, GetEligibleNodes() can just have one return parameter (i.e., nodeNames).

Reply:
Thanks, done.